src/app/Http/Requests/EmailVerificationRequest.php

```diff
@@ -3,11 +3,20 @@
 namespace Backpack\CRUD\app\Http\Requests;
 
 use Illuminate\Foundation\Auth\EmailVerificationRequest as OriginalEmailVerificationRequest;
+use Illuminate\Support\Facades\Cookie;
 
 class EmailVerificationRequest extends OriginalEmailVerificationRequest
 {
     public function user($guard = null)
     {
-        return parent::user(backpack_guard_name());
+        return parent::user(backpack_guard_name()) ?? $this->getUserFromCookie();
+    }
+
+    private function getUserFromCookie()
+    {
+        if (Cookie::has('backpack_email_verification')) {
+            return config('backpack.base.user_model_fqn')::where('email', Cookie::get('backpack_email_verification'))->first();
+        }
+        return null;
```

Comment:
Hmm I see we have these two functions, to get the user and get the user from cookie in two places - the `VerifyEmailController` and `EmailVerificationRequest`. Any chance we can have it in just one? 

As far as I can tell, it doesn't depend on anything, so I guess we can make it `static` and call it from the other one, right?